codes/zig/chapter_array_and_linkedlist/linked_list.zig

```diff
@@ -1,84 +1,103 @@
 // File: linked_list.zig
 // Created Time: 2023-01-07
-// Author: codingonion (coderonion@gmail.com)
+// Author: codingonion (coderonion@gmail.com), CreatorMetaSky (creator_meta_sky@163.com)
 
 const std = @import("std");
-const inc = @import("include");
+const utils = @import("utils");
 
 // 在链表的节点 n0 之后插入节点 P
-pub fn insert(n0: ?*inc.ListNode(i32), P: ?*inc.ListNode(i32)) void {
-    var n1 = n0.?.next;
-    P.?.next = n1;
-    n0.?.next = P;
+pub fn insert(comptime T: type, n0: *utils.ListNode(T), P: *utils.ListNode(T)) void {
```

Comment:
Is there a way to import the utils like `from utils import ListNode` so that we can access the class using `ListNode` instead of `utils.ListNode`?

Reply:
Is it recommend using `i32` instead of the generic type `T` for simplicity.